internal/dumper/dumper_test.go

```diff
@@ -114,14 +125,13 @@ func TestDumper(t *testing.T) {
 			{Name: "Extra", Type: querypb.Type_VARCHAR},
 		},
 		Rows: [][]sqltypes.Value{
-			{
-				sqltypes.MakeTrusted(querypb.Type_VARCHAR, []byte("not_deleted")),
-				sqltypes.MakeTrusted(querypb.Type_BLOB, []byte("int")),
-				sqltypes.MakeTrusted(querypb.Type_VARCHAR, []byte("YES")),
-				sqltypes.MakeTrusted(querypb.Type_BINARY, []byte("")),
-				sqltypes.MakeTrusted(querypb.Type_NULL_TYPE, []byte("NULL")),
-				sqltypes.MakeTrusted(querypb.Type_VARCHAR, []byte("VIRTUAL GENERATED")),
-			},
```

Comment:
Previously this stub's purpose was only for filtering out the single generated column. Now it's being used for grabbing all the column names.